### PR TITLE
creating view mode

### PR DESCRIPTION
**Pull request by TiFurt**

### CODE REVIEW

File: src/app/car-lift/car-lift.component.html [hunk 1]
```diff
@@ -1,171 +1,177 @@
-<mat-card class="formula-inputs" *ngIf="formGroup">
-  <mat-card-content class="values-container">
-    <div class="column">
-      <h2>Cálculo pressão tubo da seta</h2>
+<div [ngStyle]="{ 'font-size': 1 * fontScale + 'rem' }">
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.html]: we can add the font size to the body tag instead and convert all the other font sizes to em, that way we only change one place

File: src/app/car-lift/car-lift.component.ts [hunk 2]
```diff
@@ -132,6 +136,14 @@ export class CarLiftComponent implements OnInit, OnDestroy {
     this.setListeners();
   }
 
+  private listenFullScreen(): void {
+    this.fullScreenService.fullscreen$
+      .pipe(takeUntil(this._destroyed$))
+      .subscribe((isFullScreen: boolean) => {
+        this.fontScale = isFullScreen ? 1.5 : 1;
```
Comment (biel-correa) [inline src/app/car-lift/car-lift.component.ts]: add 1.5 and 1 to fullScreenService as readonly values and use it on line 34

File: src/app/core/navbar/navbar.component.ts [hunk 3]
```diff
@@ -1,17 +1,111 @@
...
+  setIcon(): void {
+    if (this.element) {
+      this.element.nativeElement
+        .querySelector('.mdc-switch__icon--on')
+        .firstChild.setAttribute('d', VIEW_MODE.ON);
+      this.element.nativeElement
+        .querySelector('.mdc-switch__icon--off')
+        .firstChild.setAttribute('d', VIEW_MODE.OFF);
+    }
+  }
```
Comment (biel-correa) [inline src/app/core/navbar/navbar.component.ts:110]: why do you need all this?

File: src/app/shared/services/full-screen.service.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FullScreenService {
+  fullscreen$ = new Subject<boolean>();
```
Comment (biel-correa) [inline src/app/shared/services/full-screen.service.ts]:
```suggestion
  fullscreen$ = new BehaviourSubject<boolean>();
```

File: src/app/core/navbar/navbar.component.ts (hunk 3 above)
Comment (TiFurt) [reply src/app/core/navbar/navbar.component.ts:110]: This is for changing the icon of mat slide toggle

File: src/app/car-lift/car-lift.component.ts (hunk 2 above)
Comment (TiFurt) [reply src/app/car-lift/car-lift.component.ts]: done

File: src/app/shared/services/full-screen.service.ts (hunk 4 above)
Comment (TiFurt) [reply src/app/shared/services/full-screen.service.ts]: done

File: src/app/car-lift/car-lift.component.html (hunk 1 above)
Comment (TiFurt) [reply src/app/car-lift/car-lift.component.html]: done

**Review by biel-correa — APPROVED** [review]

**Pull request merged**